docs/en/user-guide/persistent-subscription.md

```diff
@@ -0,0 +1,104 @@
+# Persistent and automatically update subscriptions
+
+[systemd.timer](https://www.freedesktop.org/software/systemd/man/latest/systemd.timer.html) allows you to run a service periodically.
+
+The current behavior of dae is to pull subscriptions at startup, but this can sometimes result in an empty group if there is a network anomaly or other possible reason when pulling subscriptions.
+
+We'll introduce a way to achieve persistent subscription storage and automatic subscription updates through the systemd service and timer.
+
+## Preparations
+
+### Shell script
+
+We assume that your dae configuration file is stored in `/usr/local/etc/dae/` .
+
+`/usr/local/bin/update-dae-subs.sh`:
+
+```sh
+#!/bin/bash
+
+# Change the path to suit your needs
+cd /usr/local/etc/dae || exit 1
+while IFS=':' read -r name url
+do
+    curl -o "${name}.sub" "$url"
```

Comment:
What about curl into a .sub.new and mv it to .sub if $? is 0 to avoid exploding?

Reply:
seems good